recipes/OpenVisus/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+zlib-devel curl httpd.x86_64 httpd-devel.x86_64
```

Comment:
@spetruzza I don't think conda-forge will let us go with this. All dependencies MUST come from conda-forge/conda

Reply:
Ok, reading the scripts I saw that you could include requirements for yum so I thought it was allowed. Then, I'll try to avoid that